### PR TITLE
Fixes : #784 Adds Forgot password option

### PR DESCRIPTION
**Pull request by saketkumar95**
Demo : 

demo

### CODE REVIEW

File: src/start/PasswordAuthView.js [hunk 1]
```diff
@@ -109,6 +112,14 @@ class PasswordAuthView extends React.Component {
           onPress={this.validateForm}
         />
         <ErrorMsg error={error} />
+        <Touchable>
```
Comment (vishwesh3) [inline src/start/PasswordAuthView.js]: Do we want whole area to be touchable? We want send `onPress` prop to `Touchable`, right?

File: src/start/PasswordAuthView.js [hunk 2]
```diff
@@ -109,6 +112,14 @@ class PasswordAuthView extends React.Component {
...
+          <Text style={[styles.field, styles.link]} onPress={() => openLink(forgotPasswordLink)}>
+            Forgot Password?
+          </Text>
+          <Text style={[styles.field, styles.link]} onPress={() => openLink(registerLink)}>
```
Comment (vishwesh3) [inline src/start/PasswordAuthView.js]: Use `Label` to support i18n.

**Comment by borisyankov** [comment]
Test with keyboard shown, and see there kinda isn't enough space for two links like this.

**Comment by saketkumar95** [comment]
@borisyankov This is when the keyboard is shown up! 👍 

demo

File: src/start/PasswordAuthView.js (hunk 1 above)
Comment (saketkumar95) [inline src/start/PasswordAuthView.js]: yeah! Didn't thought for that! 👍

File: src/start/PasswordAuthView.js (hunk 2 above)
Comment (saketkumar95) [inline src/start/PasswordAuthView.js]: okay! 👍

**Comment by borisyankov** [comment]
Rebase against latest master, there was a bug with the 'Sign in with Google' button not showing.

**Comment by saketkumar95** [comment]
@borisyankov Here is the demo with latest master : 

demo

I'm thinking of adding these links after the Sign in with google button

File: src/start/PasswordAuthView.js (hunk 1 above)
Comment (borisyankov) [inline src/start/PasswordAuthView.js]: You can also add an item to messages_en.json for the new strings (and skip the other translations as they are generated by Transifex)

Comment (saketkumar95) [inline src/start/PasswordAuthView.js]: okay doing that in another PR.

**Pull request merged**